packaging/docker/misc/tini-amd64.sha256sum

```diff
@@ -0,0 +1 @@
+93dcc18adc78c65a028a84799ecf8ad40c936fdfc5f2a57b1acda5a8117fa82c  tini-amd64
```

Comment:
We were previously confirming the web server didn't change the binary out from under us by fetching this file each time.  (Which mostly defeats the purpose of checking the SHA.)